src/com/content-event/event.js

```diff
@@ -85,6 +85,16 @@ export default class ContentEvent extends Component {
 			ShowGame = <ContentCommonNavButton href={path+'/games'} class={Class}><SVGIcon>gamepad</SVGIcon><div class="if-sidebar-inline">Games</div></ContentCommonNavButton>;
 		}
 
+		let ShowMyGraded = null;
```

Comment:
Inconsistent with other code. Prefer `ShowMyGrades` to `ShowMyGraded`.

Reply:
Fixed